puppet_fixtures.gemspec

```
@@ -13,4 +13,6 @@ Gem::Specification.new do |s|
   s.homepage              = 'https://github.com/voxpupuli/puppet_fixtures'
   s.metadata              = { 'source_code_uri' => 'https://github.com/voxpupuli/puppet_fixtures' }
   s.required_ruby_version = '>= 2.7', '< 4'
+
+  s.add_dependency 'rake', '~> 13.2'
```

Technically it's a soft dependency, but it may be good enough given most uses will be via rake anyway. I'd just be more open
```suggestion
  s.add_dependency 'rake', '~> 13.0'
```